### DISCUSSION TITLE
[v8] enhance testing

### DISCUSSION THREAD

**Discussion by oeddyo** [TanStack/table#4053] [General]
Hi @tannerlinsley 

not sure if this fits as in issues so want to bring up here

I noticed there are some tests but we probably can benefit from more tests given how tanstack/table is generally used across many enterprise tools. It might help we reduce small bugs here and there. I took a look but didn't see how each single functionality fits the folder structure for now as only the integration tests are there

whats your thoughts on this? @tannerlinsley

**Comment by KevinVandy** [comment]
Are you interested in contributing some tests?

**Reply by oeddyo** [reply]
yep. I contributed some code but wasn't sure where to put the tests thus the ask :)

see discussion here: https://github.com/TanStack/table/pull/4031
